flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlToOperationConverterTest.java

```diff
@@ -1660,6 +1665,65 @@ public void testReset() {
         assertThat(((ResetOperation) operation2).getKey()).hasValue("test-key");
     }
 
+    @Test
+    public void testHelpCommands() {
+        ExtendedParser extendedParser = new ExtendedParser();
+        Stream.of("HELP", "HELP;", "HELP ;", "HELP\t;", "HELP\n;")
+                .forEach(
+                        command -> {
+                            Operation operation1 =
+                                    extendedParser
+                                            .parse(command)
+                                            .orElseThrow(
+                                                    () ->
+                                                            new RuntimeException(
+                                                                    "Fail to parse '"
+                                                                            + command
+                                                                            + "'"));
+                            assertThat(operation1).isInstanceOf(HelpOperation.class);
+                        });
+    }
+
+    @Test
+    public void testClearCommands() {
+        ExtendedParser extendedParser = new ExtendedParser();
+        Stream.of("CLEAR", "CLEAR;", "CLEAR ;", "CLEAR\t;", "CLEAR\n;")
+                .forEach(
+                        command -> {
+                            Operation operation1 =
+                                    extendedParser
+                                            .parse(command)
+                                            .orElseThrow(
+                                                    () ->
+                                                            new RuntimeException(
+                                                                    "Fail to parse '"
+                                                                            + command
+                                                                            + "'"));
+                            assertThat(operation1).isInstanceOf(ClearOperation.class);
+                        });
+    }
+
+    @Test
+    public void testQuitCommands() {
+        ExtendedParser extendedParser = new ExtendedParser();
+        Stream.of(
```

Comment:
Please create a parametrized test for this. It's quite easy in JUnit 5, just use `@ValueSource`: https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-ValueSource

Reply:
I gave a quick look to this class, porting to junit5 should be as easy as just replacing the imports. Also, I think it's possible for JUnit 4 and 5 to coexist in the same class (but i'm not 100% sure)

Reply:
done